tests/management/permission/test_view.py

```diff
@@ -127,6 +127,53 @@ def test_get_list_is_the_only_valid_method(self):
         response = client.delete(url, **self.headers)
         self.assertEqual(response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
 
+    def test_filters_multiple_appliation_values(self):
+        """Test that we can filter permissions with multiple application values."""
+        with tenant_context(self.tenant):
+            expected_permissions = list(Permission.objects.values_list("permission", flat=True))
+
+        url = reverse("permission-list")
+        url = f"{url}?application=rbac,acme"
+        client = APIClient()
+        response = client.get(url, **self.headers)
+        response_permissions = [p.get("permission") for p in response.data.get("data")]
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data.get("data")), 4)
+        self.assertCountEqual(expected_permissions, response_permissions)
+
+    def test_filters_multiple_resource_type_values(self):
+        """Test that we can filter permissions with multiple resource_type values."""
+        with tenant_context(self.tenant):
+            expected_permissions = list(Permission.objects.values_list("permission", flat=True))
+
+        url = reverse("permission-list")
+        url = f"{url}?resource_type=roles,*"
+        client = APIClient()
+        response = client.get(url, **self.headers)
+        response_permissions = [p.get("permission") for p in response.data.get("data")]
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data.get("data")), 4)
```

Comment:
I am wondering if the number would cause any problem for smoke tests, where the permission data would be overwritten by actual ones, so the number might be large.

Reply:
Not sure I follow, @astrozzc? The permission data in these unit tests is isolated to the `setUp` block above, where we explicitly create 4 permissions. Smoke tests don't run within the context off the unit tests for RBAC. Let me know if I'm misunderstanding something! 😄

Reply:
Ah got it! I thought it was created from the roles/permissions folder.